assets/src/edit-story/components/panels/layer/layer.js

```diff
@@ -28,7 +28,11 @@ import { getDefinitionForType } from '../../../elements';
 import useLayerSelection from './useLayerSelection';
 import { LAYER_HEIGHT } from './constants';
 
-const LayerButton = styled.button.attrs({ type: 'button' })`
+const LayerButton = styled.button.attrs({
+  type: 'button',
+  tabIndex: '0',
```

Comment:
This is not strictly necessary.

Reply:
It could even be set to -1, if we don't want it to participate in tab order, but still be clickable with the mouse. If you're not using the mouse, there's no need to click the layers in the panel, as clicking elements on stage will have the same effect.

Reply:
I initially missed that it was a button. Could you clarify why? Do we get anything valuable? Will accessibility be confusing for a button?

Reply:
It's clickable, so a button seemed like the semantic choice here. But `role="listbox"` also semantically means clickable, so it could be a `<li>` in a `<ul>`.

It does mean we get keyboard interaction for free though, as you can tab to a layer and click <kbd>space</kbd> to select it (as space invokes `onClick` natively on buttons, but not other _faux_-interactive elements). But we probably want to remove the layers from the tab order anyway at some point, so that probably not even relevant.

Reply:
SG